packages/sitecore-jss-angular-schematics/package.json

```diff
@@ -7,7 +7,9 @@
   "sideEffects": false,
   "scripts": {
     "build": "tsc -p tsconfig.json && tsc -p tsconfig-esm.json",
-    "test": "npm run build && jasmine src/**/*_spec.js"
+    "test-package": "jasmine src/**/*_spec.js",
+    "test": "npm run build && npm run test-package",
+    "coverage": "nyc npm run test-package"
```

Comment:
This package has a slightly diferent structure for the tests scripts than the others. Just curious as to why this is.

Reply:
Because you can't get coverage in case if you run `npm run build` in the `test` command